lib/api/model/events.dart

```diff
@@ -61,6 +61,7 @@ sealed class Event {
           default: return UnexpectedEvent.fromJson(json);
         }
       // case 'muted_topics': … // TODO(#422) we ignore this feature on older servers
+      case 'user_status': return UserStatusEvent.fromJson(json);
```

Comment:
```
api: Add user_status event
```

`tools/check build_runner` is failing at this commit; I think that can be fixed with `tools/check build_runner --fix`.